gce-staging-registry-1/main.tf

```diff
@@ -0,0 +1,56 @@
+variable "env" {
+  default = "staging"
+}
+
+variable "gce_gcloud_zone" {}
+
+variable "index" {
+  default = 1
+}
+
+variable "project" {
+  default = "eco-emissary-99515"
+}
+
+variable "syslog_address_com" {}
+variable "syslog_address_org" {}
+
+variable "travisci_net_external_zone_id" {
+  default = "Z2RI61YP4UWSIO"
+}
+
+terraform {
+  backend "s3" {
+    bucket         = "travis-terraform-state"
+    key            = "terraform-config/gce-registry-staging-1.tfstate"
+    region         = "us-east-1"
+    encrypt        = "true"
+    dynamodb_table = "travis-terraform-state"
+  }
+}
+
+provider "google" {
+  project = "${var.project}"
+  region  = "us-central1"
+}
+
+data "terraform_remote_state" "registry_staging_1" {
```

Comment:
I don't see this being used, and I don't expect you will need to pull in the remote state of the same graph that is being built here.